integreat_cms/cms/templates/hix_widget.html

```diff
@@ -0,0 +1,45 @@
+{% load i18n %}
+
+<!--HIX widget-->
+<div data-content-changed class="rounded border border-blue-500 shadow-2xl bg-white">
+    <div class="rounded p-4 bg-water-500">
+        <h3 class="heading font-bold text-black">
+            {% trans "HIX Value" %}
+        </h3>
+    </div>
+    <div class="w-full px-4 pb-4 rounded shadow bg-white">
+        <div class="relative h-full">
+            <div id="hix-loading" class="hidden absolute w-full h-full bg-white bg-opacity-75">
+                <div class="relative top-1/3 m-auto w-1/3"><i data-feather="loader" class="animate-spin"></i> {% trans 'Loading...' %}</div>
+            </div>
+            <label>{% trans 'Improve the understandability of the page content.' %}</label>
+            <canvas id="hix-chart"></canvas>
+            <div class="mb-2">
+                <div data-hix-state="updated" class="bg-green-400 rounded p-2 hidden">
+                    <i data-feather="check-circle"></i>
+                    {% trans 'HIX value is up to date.' %}
+                </div>
+                <div data-hix-state="outdated" class="bg-yellow-400 rounded p-2 hidden">
+                    <i data-feather="alert-circle"></i>
+                    {% trans 'HIX value is outdated.' %}
+                </div>
+                <div data-hix-state="error" class="bg-red-400 rounded p-2 hidden">
+                    <i data-feather="x-circle"></i>
```

Comment:
If there is an "x"-button, I think it should have a function.

Reply:
I have implemented the changes locally, but I did not push them yet, because the pr is blocked on 27d84e11e26a2c1b04bf6758ef70f6b07136a004 now. Should I push my local changes anyways?